plugins/pipelines/README.md

```diff
@@ -121,9 +121,7 @@ Only PR events of specified PR number will be searched when `prNum` is set
 #### Get all stages for a single pipeline
 
 `GET /pipelines/{id}/stages`
-Will get latest commit event's stages if no event ID or group event ID is provided
-
-`GET /pipelines/{id}/stages?groupEventId={groupEventId}`
+Will get latest commit event's stages if no event ID is provided
```

Comment:
With the latest schema changes, `stages` are no longer associated to `event`. I believe `GET /pipelines/{id}/stages?eventId={eventId}` is no longer valid endpoint.
```suggestion
```